content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -52,7 +52,7 @@ Memory is reported as the working set, in bytes, at the instant the metric was c
 [Metrics Server](https://github.com/kubernetes-incubator/metrics-server) is a cluster-wide aggregator of resource usage data.
 It is deployed by default in clusters created by `kube-up.sh` script
 as a Deployment object. If you use a different Kubernetes setup mechanism you can deploy it using the provided
-[deployment yamls](https://github.com/kubernetes-incubator/metrics-server/tree/master/deploy).
+[deployment component.yaml](https://github.com/kubernetes-sigs/metrics-server/releases) file.
```

Comment:
@laurenhughes The manifest you need is actually called `components.yaml` (with an “s”), right?

Reply:
@sftim Yes, thanks for the catch!